sdcm/mgmt/cli.py

```diff
@@ -23,7 +23,8 @@
 from distutils.version import LooseVersion
 
 import requests
-from invoke.exceptions import UnexpectedExit, Failure
+from invoke.exceptions import UnexpectedExit as InvokeUnexpectedExit, Failure as InvokeFailure
```

Comment:
I think you can remove `UnexpectedExit`, since in both cases in inherit from `Failure`, catching Failure would be enough.

Reply:
That's correct. Done.